### ISSUE TITLE
There are many duplicates in the recognition results

### ISSUE THREAD

**Issue by yangsuxia** [sanchit-gandhi/whisper-jax#76]
Can you help me with the following two questions when using Whisper jax？

1.  I have a long audio of 1minutes, and during decoding, there was a sentence that repeated many times. However, when importing this audio on the huginface, this situation did not occur during decoding. Why？

![image](https://user-images.githubusercontent.com/34536059/236426111-d55c8c0f-4172-404b-bd4b-b184b154f2ec.png)

2.  Which model was used as the benchmark in the report, and what is the batch size

**Comment by yangsuxia** [comment]
> Hey @yangsuxia - you should use `return_timestamps=True` in this case as it greatly reduces model hallucinations. Regarding benchmarks, see #5.

Yes, after setting the parameter  return_timestamps to true, the number of repetitions has decreased significantly, but there are still

I would like to ask another question and look forward to your reply

I use large_ V2 model, batch_ Size=16, dtype=jnp.float16, decoding audio for one minute takes about 2.2s, is it normal？

**Comment by RaulKite** [comment]
Same problem here. I have tried with large-v2  and large models and there are too many duplicates, even with return_timestamps=True

**Comment by flexchar** [comment]
I am researching for ways to post clean this without using LLMs. If anyone has come across any ideas, I would appreciate them!
